crml/erc20-peg/contracts/ERC20Peg.sol

```diff
@@ -28,7 +28,7 @@ contract ERC20Peg is Ownable {
         require(depositsActive, "deposits paused");
 
         // CENNZ deposits will require a vote to activate
-        if (address == 0x1122b6a0e00dce0563082b6e2953f3a943855c1f) {
+        if (address(0) == 0x1122B6a0E00DCe0563082b6e2953f3A943855c1F) {
```

Comment:
used https://ethsum.netlify.app/ to get correct checksum

Reply:
```suggestion
        if (address(tokenType) == 0x1122B6a0E00DCe0563082b6e2953f3A943855c1F) {
```